tests/algorithms/adversarial/test_algorithms.py

```diff
@@ -124,18 +77,19 @@ def trainer(
     else:
         expert_data = expert_transitions
 
-    venv = util.make_vec_env(
-        "seals/CartPole-v0",
-        n_envs=1,
-        parallel=_parallel,
-        log_dir=tmpdir,
-    )
-
-    _algorithm_cls = _algorithm_kwargs.pop("algorithm_cls")
-    gen_algo = util.init_rl(venv, verbose=1, **_algorithm_kwargs)
+    algorithm_kwargs = dict(algorithm_kwargs)
+    _algorithm_cls = algorithm_kwargs.pop("algorithm_cls")
+    venv = util.make_vec_env(env_name, n_envs=num_envs, parallel=parallel)
```

Comment:
Did you remove `log_dir=tmpdir` deliberately here?

Reply:
I didn't, but I think it's actually fine to remove -- for `make_vec_env` it defaults to `None`, which will not log Monitor stats to any directory, which seems fine for testing.